frontend/test/metabase/scenarios/admin/databases/list.cy.spec.js

```diff
@@ -6,21 +6,30 @@ describe("scenarios > admin > databases > list", () => {
     cy.signInAsAdmin();
   });
 
-  it.skip("should not display error messages upon a failed `GET` (metabase#20471)", () => {
+  it("should not display error messages upon a failed `GET` (metabase#20471)", () => {
     const errorMessage = "Lorem ipsum dolor sit amet, consectetur adip";
 
-    cy.intercept("GET", "/api/database", req => {
-      req.reply({
-        statusCode: 500,
-        body: { message: errorMessage },
-      });
-    }).as("failedGet");
+    cy.intercept(
+      {
+        method: "GET",
+        pathname: "/api/database",
+        query: {
+          exclude_uneditable_details: "true",
+        },
+      },
+      req => {
+        req.reply({
+          statusCode: 500,
+          body: { message: errorMessage },
+        });
+      },
+    ).as("failedGet");
 
     cy.visit("/admin/databases");
 
     cy.wait("@failedGet");
     // Not sure how exactly is this going the be fixed, but we should't show the full error message on the page in any case
-    cy.findByText(errorMessage).should("not.exist");
+    cy.findByText(errorMessage).should("not.be.visible");
```

Comment:
The fix is that I wrap the error message inside a collapsable UI, so it'll always exist, just isn't visible.